src/styles/step.css

```diff
@@ -0,0 +1,19 @@
+.step-number {
```

Comment:
Before you get too far into the styling, one of the requirements was to use a css pre-processor. You may want to bring it in early in the project.